NF.csv

```diff
@@ -601,6 +600,17 @@ Ganglioglioma,GNG,,,FALSE,ONCOTREE,tumorType,,,,,,,,,,
 Anaplastic Ganglioglioma,AGNG,,,FALSE,ONCOTREE,tumorType,,,,,,,,,,
 Optic Pathway Glioma,A glioma that affects the optic nerve. This condition can be seen in association with neurofibromatosis 1. It is most commonly seen in the pediatric age group. ,,,FALSE,https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C4537,tumorType,,,,,,,,,,
 Not Applicable,,,,FALSE,Sage Bionetworks,tumorType,,,,,,,,,,
+Colorectal Carcinoma,A malignant epithelial neoplasm that arrises from the colon or rectum and invates through the muscularis mucosa into the submucosa.,,,FALSE,obo:NCIT_C2955,tumorType,,,,,,,,,,
+Recurrent MPNST,The reemergence of malignant peripheral nerve sheath tumor after a period of remission.,,,FALSE,obo:NCIT_C8823,tumorType,,,,,,,,,, 
+Juvenile Myelomonocytic Leukemia,A myelodysplastic/myeloproliferative neoplasm of childhood that is characterized by proliferation principally of the granulocytic and monocytic lineages,,,FALSE,obo:NCIT_C9233,tumorType,,,,,,,,,,
```

Comment:
JMML is captured in ONCOTREE, could we use that instead? 

Also, it looks like oncotree is missing the NCIT mapping for JMML...I will raise an issue

Reply:
NCIT is a great resource and I definitely default to it for most biomedical data, but (IMO) ONCOTREE has a much more well defined tumor hierarchy and semantics, whereas NCIT has a lot of tumor terms that are overlapping. From a practical perspective, ONCOTREE will also be required to upload data to cBioPortal. 

At the end of the day, I think it's fine to go with one or the other, but if we switch to NCIT for these terms we should probably change the other ONCOTREE terms in our dictionary to NCIT, and we should also be sure to use NCIT calues that can be translated to ONCOTREE (https://github.com/cBioPortal/oncotree/blob/master/scripts/ontology_to_ontology_mapping_tool/ontology_mappings.txt)